tests/runner.rs

```diff
@@ -320,13 +320,13 @@ fn ciphers() {
 #[test]
 #[ignore]
 fn server() {
-    let (port, port_str) = choose_port();
-
     fn curl(port: u16) {
         Command::new("curl")
             .env("LD_LIBRARY_PATH", "")
             .args([
                 "-v",
+                "--header",
+                "Host: localhost:port",
```

Comment:
What does this do? Seems surprising to have `port` literally here?

Reply:
Removed the `:port` for clarity. Because tests/server.c dumps all the data it receives from curl, and because we check its output matches (whether run against openssl or rustls libssl), we need the data sent by curl to be determinstic. This commit stopped that being true, because the server port now varies between runs.